docs/drivers.rst

```diff
@@ -287,6 +287,7 @@ Other
     Character LCD <https://circuitpython.readthedocs.io/projects/charlcd/en/latest/>
     HT16K33 LED Matrices and Segment Displays <https://circuitpython.readthedocs.io/projects/ht16k33/en/latest/>
     IS31FL3731 Charlieplexed LED Matrix <https://circuitpython.readthedocs.io/projects/is31fl3731/en/latest/>
+    IS31FL3741 RGB LED Matrix driver <https://circuitpython.readthedocs.io/projects/is31fl3741/en/latest/>
```

Comment:
Do both of these lines belong here? they point to the same URL?

Reply:
They differ in one character:
circuitpython.readthedocs.io/projects/is31f**l3731**/en/latest/
circuitpython.readthedocs.io/projects/is31fl**3741**/en/latest/

Reply:
One is 3731, the second is 3741. The new one is one digit off.